draft-quic-atsss-reqs.mkd

```diff
@@ -497,6 +498,9 @@ same ATSSS UPF will not necessarily be handled by the same server.
 A third solution is to leverage the ability of QUICv1 to support multiple
 streams and the Unreliable Datagram Extension but extend is with
 Multipath capabilities as described in {{I-D.deconinck-quic-multipath}}.
+In addition, {{I-D.an-multipath-quic-traffic-distribution}} describes several 
```

Comment:
I read again your draft. It contains three main sections: packet scheduler, stream management and congestion control. In these three sections, it indicates that there are some open issues but does not provide a clear recommendation.
For packet scheduling, I would prefer to point to references showing that there are solutions that work for packet schedulers such as MPTCP papers/drafts and the iccrg draft that describes abstract schedulers.
For stream management, I would prefer to avoid opening this problem since this could refer to HTTP priorities that some IETFers do not consider as solved.
For congestion control, there are multiple references in MPTCP with LIA, OLIA or BALIA showing that this is a solved problem.

The main message in this part of the document should be that many of these issues have already been addressed and solved by the MPTCP community and thus do not require additional research.

Reply:
> For packet scheduling, I would prefer to point to references showing that there are solutions that work for packet schedulers such as MPTCP papers/drafts and the iccrg draft that describes abstract schedulers.

Thanks for the clarification. I will add some related references.

Reply:
We will update the draft-iccrg-schedulers that documents existing schedulers in abstract terms. This is an informational draft, I don't think that 3GPP needs a draft that defines schedulers, but maybe I'm wrong

Reply:
I think the purpose of listing some existing scheduler solutions of MPTCP and MPQUIC, is to state that there is necessity to include MPQUIC standardization in IETF, in order to provide a consensus-based solution to industry. Therefore I think it's unharmful to include those references.

Reply:
I would personally prefer listing schedulers in the draft-iccr-schedulers than here

Reply:
The source for draft-schedulers is at https://github.com/obonaventure/draft-schedulers

Reply:
> I should point out that the more references we make to work that is not either currently chartered or under discussion in an IETF working group, the more likely it is that IETF people can decide that ATSSS work is someone else's problem

Fully agree with Spencer.

Reply:
> I would personally prefer listing schedulers in the draft-iccr-schedulers than here

I can live with it.